### ISSUE TITLE
How to select BAM files for contamination estimation?

### ISSUE THREAD

**Issue by StuYuXu** [grenaud/schmutzi#21]
Hello, this is very good software. Thanks to you for developing it. However, I have a question, and I hope to get your help.

As the principle of your software, there are two parts: one performing damage-based contamination estimation and the other utilizing a database of mitochondrial allele frequencies.

So, does that mean I need one original bam (contained terminal damage information bam) for the first step? Then I use the trimmed bam (cut 10 bp to remove terminal damage bam) for the second step.

As I noticed, if I only used original bam (contained terminal damage information bam) for these two steps, the consensus sequence of my data was called contained terminal damage information. Thus, might it influence the results?

So, I hope you can help me with this problem!

Student,

Yu Xu

**Comment by grenaud** [comment]
schmutzi handles damage internally and this is what powers the inference. You do not need to trim anything just map your ancient DNA fragments to the mitochondrial genome in isolation

**Comment by StuYuXu** [comment]
Thanks for you sincerely response.
I successfully used your software to get the contamination estimation.

`contDeam.pl --library double --out sample_1 sample_1.mapped.bam_MT.MD.bam`
`schmutzi.pl --t 30 --qual 30 --uselength --ref hs37d5_MT.fa sample_1 ./eurasian/freqs/ sample_1.mapped.bam_MT.MD.bam`

**Comment by grenaud** [comment]
great, but beware that contDeam only gives you a ballpark estimate and there are a lot of assumptions in there. the iterative approach is the way to go :-)

**Comment by Shaileshdesai76** [comment]
sorry for adding comment on this threads but my question is related to this. and i am bit new to ancient DNA field. 
My organims is non-model and it doesnt have freq file and my library were UDG treated, can i direclty use the endoCaller for consensus calling ??
i understand also that most people use schmutzi for contamination estimation, but i found this straight forward to make consensus calling so using it. 
I look forward for your suggestions.

**Comment by grenaud** [comment]
@Shaileshdesai76  Yes absolutely, I would still check if you have residual damage using bam2prof. endoCaller is designed to call a consensus with or without damage.

**Comment by Shaileshdesai76** [comment]
This is great.
Thank you for clarification.


*Shailesh Desai*
*PHD Student*

*Cytogenetic lab, *
*Dept.**, of Zoology, Institute of Science,*
*BHU, Varanasi. *
Cell -      (+91) 7698946714
